pacai/util/mazeGenerator.py

```diff
@@ -19,6 +19,7 @@
 @author: Jie Tang
 """
 
+import logging
 import random
```

Comment:
There are a ton of places in this file that use the incorrect random (i.e. they do not use the rng you created, they use `random.*`).

Reply:
Make sure to respond to all comments.